menpodetect/opencv/detect.py

```diff
@@ -14,6 +14,13 @@
 from .conversion import (pointgraph_from_rect, opencv_frontal_face_path,
                          opencv_profile_face_path, opencv_eye_path)
 
+def _get_appropiate_flags():
+    from sys import version_info
+    if version_info.major == 2:
```

Comment:
This is great, thanks @grigorisg9gr. However, rather than checking the Python version, can we check the OpenCV version? Something like ``cv2.__version__.split('.')[0] == '2'`` to check for OpenCV2. This will be more robust, hopefully.